src/pages/develop/services.md

```diff
@@ -78,3 +76,7 @@ You can see services metrics under the [Metrics] tab of the service view.
 ## Deleting a Service
 
 You can delete your service by opening the project's settings and navigating to the danger page.
+
+## Service Icons
+
+Within the Command + K context menu, you can change the default service icon to something else. The list of available icons are pulled from our [devicons](https://devicons.railway.app/) service.
```

Comment:
```suggestion
Within the Command + K menu, you can change the default service icon to something else. If no URL to an icon is provided, we will use an image from our [devicons](https://devicons.railway.app/) service.
```

Reply:
Do we want to say command palette? Because for Windows/Linux users, it's ctrl + k.